### ISSUE TITLE
Support for tfio in tensorflow backend

### ISSUE THREAD

**Issue by lminer** [triton-inference-server/server#4162]
tfio has a lot of ops that are necessary for audio models. Is there support included for tfio in then tensorflow backend?

If not, it would be a great addition.

**Comment by tanmayv25** [comment]
I have spent a brief time on reading about the [project](https://github.com/tensorflow/io). It looks more geared towards model training instead of inference. Can you give some links to these ops? 

That being said. we have a way to provide custom ops to models loaded via tensorflow backend. See [here](https://github.com/triton-inference-server/server/issues/3604) for how we load models with ops from tensorflow-text.

**Comment by lminer** [comment]
You're right it's mostly useful for training, but for us there are ops that we would also use during inference. For example, [resample](https://www.tensorflow.org/io/api_docs/python/tfio/audio/resample), which would be used as a layer within the model itself for certain downsampling use cases.
